### DISCUSSION TITLE
[Kavita] Display Title at volume/issue level if available

### DISCUSSION THREAD

**Discussion by Roshri** [Kareadita/Kavita#2586] [Ideas]
### Idea Description

I'd like to have the ability to see individual titles as a subtitle of volumes/issues if available. The request is for list view but could also be extended for card view.

Currently, we only get Series Title - Volume/Issue xx

Ideally we could get
Volume/Issue xx
Title

If title is not present just ignore the second line and display only the numbering, as Series Title is already present at the top

### Duration of Using Kavita

a year

### Idea Category

User Experience

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by lyingloverr** [comment]
I would like this on a chapter level as well. Its fun seeing mangas like "Call of The Night" that have their chapters named "Night xyz"

**Reply by rix8675** [reply]
this would be fantastic having the title appear in place of or with the Chapter names using the cards view.
i've been using kavita for at least a year now and mainly used it for light novels and a few comics but the libraries grown and now i'm trying to input more metadata into the comics/manga and would like to compress the amount of series and put subseries into one series and have the volumes as the subseries names.

**Comment by majora2007** [comment]
This should be completed as of the (current nightly or) v0.8.3 release. For series, the series name and localized name will show on the cards. For chapter/volumes, they will show on the card. In addition, you can click these cards and view a detail page with their metadata. 

![image](https://github.com/user-attachments/assets/0987a29f-4a5c-464c-87f4-9c98afd6ac1e)
![image](https://github.com/user-attachments/assets/857cd37c-3e2f-4a58-894a-6b5f5064e404)

If this doesn't meet expectations, please comment and I will re-open the request.

**Reply by Roshri** [reply]
Correct me if I'm wrong, your screenshot is from a light novel/book library right? Will this also be available in other types of libraries?

Oh, and having publisher icons looks really neat, good work on that UI upgrade!

**Reply by majora2007** [reply]
Yes it's available across the board
